commands

```diff
@@ -35,9 +35,6 @@ case "$1" in
     ID=$(docker run --name "$SERVICE_NAME" -v "$SERVICE_ROOT/data:/data" -v "$SERVICE_ROOT/config:/usr/local/etc/redis" -d --restart always --label dokku=service --label dokku.service=redis "$PLUGIN_IMAGE:$PLUGIN_IMAGE_VERSION")
     echo "$ID" > "$SERVICE_ROOT/ID"
 
-    dokku_log_verbose_quiet "Waiting for container to be ready"
-    sleep 10
-
```

Comment:
Just switch to an env var for this. `DOKKU_SERVICE_READY_SLEEP` sounds fine to me.